src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -256,7 +252,10 @@ define(function (require, exports, module) {
                         </div>
                     </TitleHeader>
                     <div className={containerClasses}>
-                        {containerContents}
+                        <ExportList {...this.props}
```

Comment:
this is definitely a bug, probably caused by merge conflict resolution.  `containerContents` is defined above, and that is what should be inserted here.